app/services/github_service.rb

```diff
@@ -28,8 +28,8 @@ def load_repositories(access_token)
       # Add repos and remove user from any repos they no longer have access to
       remove_outdated(user, outdated_repos)
     end
-
-    def submit_issue(repo_id, sub_name, email, title, details)
+    
+    def submit_issue(repo_id, sub_name, email, email_public, title, details)
```

Comment:
@schneidmaster I couldn't find something to disable this lint. Have you something new about that?